topics/icrc_29_window_post_message_transport.md

```diff
@@ -29,9 +30,7 @@ The message sent by the relying party has `targetOrigin` set to `'*'` and is a [
 }
 ```
 
-The signer should reply with a message indicating that it is ready to receive additional messages.
-The message `targetOrigin` should be set to the previously received `icrc29_status` message its `origin` property value.
-
+The signer should send responses with the `targetOrigin` set to the previously received `icrc29_status` message its `origin` property value:
```

Comment:
ditto, no more mention of ready?

I would also maybe use singular - "The signer should respond" - instead of "responses". I was confuse if it meant that the signer should respond with multiple responses. WDYT?

Reply:
You're right, added it back.

Clarified that the signer should respond to every message.